src/eterna/AnnotationManager.ts

```diff
@@ -1111,8 +1104,7 @@ export default class AnnotationManager {
             if (relPosition && absolutePosition) {
                 // Set position
                 view.display.position.copyFrom(absolutePosition);
-                // Save display
-                this._annotationViews.push(view);
+                // pose.addAnnotationView(view, false, true);
```

Comment:
Wanted to check-in about your motivation to leave this here commented out

Reply:
That was an accident, it's removed in a newer revision